publish-packages.cmd

```diff
@@ -0,0 +1,28 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+set packagesLog=publish-packages.log
+echo Running publish-packages.cmd %* > %packagesLog%
+
+if /I [%1] == [/?] goto Usage
+if /I [%1] == [/help] goto Usage
+
+REM ensure that msbuild is available
+echo Running init-tools.cmd
+call %~dp0init-tools.cmd
+
+echo msbuild.exe %~dp0src\publish.builds %* /nologo /v:minimal /flp:v=detailed;Append;LogFile=%packagesLog% >> %packagesLog%
+call msbuild.exe %~dp0src\publish.builds %* /nologo /v:minimal /flp:v=detailed;Append;LogFile=%packagesLog%
+if NOT [%ERRORLEVEL%]==[0] (
+  echo ERROR: An error occurred while publishing packages, see %packagesLog% for more details.
+  exit /b
+)
+
+echo Done publishing packages.
+exit /b
+
+:Usage
```

Comment:
What happens if someone calls this without passing those arguments? Hopefully they get a reasonable error message. This is one aspect I don't really like about having this as part of the "dev workflow" but I'm fine with it being a step in the "official workflow"